src/applications/accredited-representative-portal/components/POARequestsTableFetcher/POARequestsTableFetcher.jsx

```diff
@@ -15,7 +15,7 @@ const POARequestsTableFetcher = ({ usePOARequests }) => {
       />
     );
   if (error) return <ErrorMessage />;
-  if (!poaRequests || poaRequests.length === 0)
+  if (!poaRequests)
```

Comment:
`poaRequests.length === 0`

Reply:
@nihil2501 checking with request length returns test errors - browser flakiness. I can restore the code to use what we were using in the first place `!poaRequests || poaRequests.length === 0` or just stick with `!poaRequests`. That doesn't raise any test errors